frontend/package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
+    "chokidar": "^2.1.8",
```

Comment:
ngano ni

Reply:
after nakog fix sa file watcher, 
kay kanang chokidar na error nigawas and need i install
mao to ako gi install, and nigana na